packages/link/src/__demo__/examples/arbitraryChildren.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { mineralTheme } from '@mineral-ui/component-utils';
+import Link from '../../Link';
+
+export default {
+  title: 'Arbitrary children',
+  description: 'A Link will render any children.',
+  scope: { Link },
+  source: `<Link to="http://example.com">
```

Comment:
This link doesn't actually work.  I click it and it changes the url to `http://0.0.0.0:8080/http://example.com` but it does not navigate.